build/run-in-docker

```diff
@@ -18,11 +18,13 @@
 
 # Run a command in a Docker container with devtoolset
 
-set -e
+set -ex
```

Comment:
We don't want this by default, do we?  run-in-docker is already verbose enough with the Docker build output.